code/game/gamemodes/blob/powers.dm

```diff
@@ -8,7 +8,7 @@
 // Power verbs
 
 /mob/camera/blob/proc/place_blob_core(var/point_rate = base_point_rate, var/override = 0)
-	if(placed)
+	if(src.placed && override != -1) //byond is a pile of dicks so I need to do unimplied src for some reason
```

Comment:
The only time it should matter is if you have a different variable for "placed" inside of a proc than the one on the mob itself. I ran into this issue once myself, before I noticed they could be different. 

Attempt to debug this by adding a world output containing both "placed" and "src.placed". See if they wind up being different somehow.